alphagov/pay-direct-debit-connector
# PP-4491: Remove chamber update script

Chamber is no longer used since native ECS secrets are now used
instead. The `update-chamber.sh` script is no longer needed.

## How to test
Ensure this is _definitely_ no longer needed. Build and deploy the application as usual.
